js/main.js

```diff
@@ -403,15 +403,8 @@
 
     }
     function loadContent(href) {
-        if(href.startsWith('/')) {
```

Comment:
This causes a chain of `/././` ... while clicking internal Markdown links. It was a poor fix: requested paths has to be verified and answered with a 404 if invalid.